drivers/wifi/nrf700x/zephyr/src/qspi/inc/ficr_prog.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @brief Header containing address/offets and functions for writing
+ * the FICR fields of the OTP memory on nRF7002 device
+ */
+
+#ifndef __OTP_PROG_H_
+#define __OTP_PROG_H_
+
+#include <stdio.h>
+#include <stdlib.h>
+
+/* REGION PROTECT : OTP Address offsets (word offsets) */
+#define     REGION_PROTECT          64
+#define     QSPI_KEY                68
+#define     MAC0_ADDR               72
+#define     MAC1_ADDR               74
+#define     CALIB_XO                76
+#define     CALIB_PDADJMCS7         77
+#define     CALIB_PDADJMCS0         78
+#define     CALIB_MAXPOW2G4         79
+#define     CALIB_MAXPOW5G0MCS7     80
+#define     CALIB_MAXPOW5G0MCS0     81
+#define     CALIB_RXGAINOFFSET      82
+#define     CALIB_TXPOWBACKOFFT     83
+#define     CALIB_TXPOWBACKOFFV     84
+#define     REGION_DEFAULTS			85
+#define     OTP_MAX_WORD_LEN		128
```

Comment:
Those alignments look messy, please fix them (same applies to other defines below). I suspect it could be editor misconfiguration on your side? We use 8 spaces/tab in NCS.